apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  styled,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioLogoFlatIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { useState, type ComponentProps } from "react";
+
+const StyledLink = styled("a", {
+  textDecoration: "none",
+  color: "white",
+  display: "inline-flex",
+  cursor: "default",
+});
+
+const PopoverItemButton = ({
+  href,
+  ...buttonProps
+}: ComponentProps<typeof Button> & { href?: string }) => (
+  <StyledLink href={href} target="_blank" referrerPolicy="no-referrer">
+    <Button
```

Comment:
why wouyld you do this? instead of just <Button as="a">

Reply:
It was not possible. I tried to put `as` as a prop but it was not there on the component.